controllers/users.controller.js

```diff
@@ -13,6 +13,21 @@ const create = async (req, res) => {
   }
 };
 
+const update = async (req, res, next) => {
+  try {
+      const id = req.params.id;
+      const updates = req.body
+      const result = await User.findByIdAndUpdate(id, updates);
+      return res.status(200).json({result});
```

Comment:
You might want your response object to have a more meaningful structure.

Let's change the object identifier to be 
```js
return res.status(200).json({ user });
```